### PR TITLE
:sparkles: Add Social Rules to CoC

### LINKED ISSUE #7: Add Recurse's Social Rules terms to OTC's CoC
[Recurse's Social Rules](https://www.recurse.com/social-rules) are important things everyone should be aware of and follow, so add the four rules to the 'Introduction' section of the Code of Conduct (CoC) page and add a link to the Social Rules page as well, both in the 'Introduction' and the 'Attribution' sections.

Credits: @tusharnankani

### LINKED ISSUE #11: Introduction and Attribution sections of CoC carry redundant info
Both sections have essentially the same info. 

image

image

Motion to strike the Introduction and start with the Pledge.

### PR DESCRIPTION
**Pull request by sreekaransrinath**
Closes #7.
(post-review) edit: Closes #11.

Adding social rules => Updating the Introduction and Attribution sections (a redundancy noted in #11), and actually adding the rules to the "Our Standards" section.

Two points to be noted:

1. My formatter went haywire, resulting in many changed lines. The contributing guidelines don't make it clear whether this is okay (see #10). If necessary, I can amend the commit to undo this.

2. I had to choose between adding a concise version of the rules (like in https://kentcdodds.com/conduct), or the explanations as well (like in https://www.recurse.com/social-rules). For the moment, I've chosen to go with the former approach. As I see it, there are a few ways to do this:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a. Leave it as it is in the commit.
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; b. Add a line linking back to https://www.recurse.com/social-rules for further explanation.
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; c. Separate the social rules into another page entirely, seeing as they're not rules set in stone in the Code of Conduct.
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; d. Include the entire explanation of all four rules on the CoC page.

I don't recommend the last option, seeing as the page is already text-heavy. Regardless, I'll amend the commit based on reviews and suggestions.

### CODE REVIEW

File: coc.html [hunk 1]
```diff
@@ -1,327 +1,288 @@
...
+			<li>Giving and gracefully accepting constructive feedback.</li>
+			<li>
+				Accepting responsibility and apologizing to those affected
+				by our mistakes and learning from the experience.
+			</li>
+			<li>
+				Focusing on what is best not just for us as individuals, but
+				for the overall community.
+			</li>
+		</ul>
```
Comment (tusharnankani) [inline coc.html:103]: Let's add a `<br/>` here?

image

File: coc.html [hunk 2]
```diff
@@ -1,327 +1,288 @@
...
+				Publishing others’ private information, such as a physical
+				or email address, without their explicit permission.
+			</li>
+			<li>
+				Other conduct which could reasonably be considered
+				inappropriate in a professional setting.
+			</li>
+		</ul>
+		<br/>
+		<p>In addition to these rules of conduct, everyone participating in events is encouraged to abide by the Recurse Center Social Rules:</p>
```
Comment (tusharnankani) [inline coc.html]: And let's hyperlink this as well.

```html
<a target="_blank" rel="noreferrer" href="https://www.recurse.com/social-rules">Recurse Center's Social Rules</a>.
```

File: coc.html [hunk 3]
```diff
@@ -1,327 +1,288 @@
...
+		<ul>
+			<li>No well-actually's</li>
+			<li>No feigning surprise</li>
+			<li>No backseat driving</li>
+			<li>No subtle -isms</li>
+		</ul>
+		<p>The social rules name subtle behaviors that put other people down or show how much we know instead of supporting each other’s learning.
+		One thing that often surprises people about the social rules is that we expect people to break them from time to time. We’re all human, and we all make mistakes. The social rules are a guide to help us be better, not a list of things we’re not allowed to do.</p>
+
```
Comment (tusharnankani) [inline coc.html]: Has prettier not impacted here?
Can you take a look?

**Review by tusharnankani — COMMENTED** [review]
Thank you so much @sreekaransrinath for this!

File: coc.html (hunk 1 above)
Comment (sreekaransrinath) [reply coc.html:103]: This is one of the formatting issues I noticed.  I have a different fix for this prepped. Let's have this PR focus only on adding the rules.

I'll create a separate PR to fix page formatting and some grammatical nitpicks I have once this is merged.

File: coc.html (hunk 2 above)
Comment (sreekaransrinath) [reply coc.html]: Pushed a new commit that should take care of this. Resolving conversation. Thanks!

File: coc.html (hunk 3 above)
Comment (sreekaransrinath) [reply coc.html]: The new commit should have fixed this. Please take a look and let me know.

File: coc.html (hunk 1 above)
Comment (tusharnankani) [reply coc.html:103]: Alright sure!

**Review by tusharnankani — APPROVED** [review]
Looks alright.
Just take a look at https://github.com/OurTechCommunity/website/issues/11#issuecomment-1279091177.

**Comment by sreekaransrinath** [comment]
Done @tusharnankani

**Comment by sreekaransrinath** [comment]
Bump @tusharnankani

**Comment by tusharnankani** [comment]
Looks good to me. Merging this!
Thank you, Sreekaran.

**Pull request merged**